build/BuildSteps.cs

```diff
@@ -134,13 +128,6 @@ public static void DotnetPublishForZips()
             }
         }
 
-        private static string BuildNet8ArtifactFullPath(string runtime)
-        {
-            return Path.Combine(Settings.OutputDir, BuildNet8ArtifactDirectory(runtime));
-        }
-
-        private static string BuildNet8ArtifactDirectory(string runtime) => $"{runtime}{Net8ArtifactNameSuffix}";
-
         private static void ExecuteDotnetPublish(string outputPath, string rid, string targetFramework, bool skipLaunchingNet8ChildProcess)
```

Comment:
Could we remove the code path which relies on `skipLaunchingNet8ChildProcess`? (here and the CLI project) I do not think that is needed anymore since this branch is specifically for oop use cases.